dapps/profile.vue/src/components/wallet/wallet.vue

```diff
@@ -22,6 +22,7 @@ https://evan.network/license/ */
             @click="activeMode = 1; $store.state.uiState.swipePanel = 'sendEve'"
             style="min-width: 250px"
             type="secondary"
+            :disabled="balance < 1.0"
```

Comment:
```suggestion
            :disabled="balance < 0.1"
```

Sending less than 1 eve is possible, the transaction fee is only 0.001 EVE. I would suggest to reduce it to 0.1.